src/inmanta/tracing.py

```diff
@@ -58,11 +58,11 @@ def configure_logfire(service: str) -> None:
 
         AsyncPGInstrumentor(capture_parameters=detailed_reporting).instrument()
 
+        logfire.instrument_pydantic("all" if detailed_reporting else "off")
         logfire.configure(
             service_name=service,
             send_to_logfire="if-token-present",
             console=False,
-            pydantic_plugin=logfire.integrations.pydantic.PydanticPlugin(record="all") if detailed_reporting else None,
```

Comment:
I followed this "migration" guide: https://github.com/pydantic/logfire/pull/453/files#diff-81de9da67fe3ff09b76f8fdf41362eac96ddeb4fe8e3de76d53cab0cf3e1ba70L3-R3